automations/data/labels.yml

```diff
@@ -82,6 +82,14 @@ groups:
         description: Bug fix
         emoji: "🛠"
 
+  - name: tooling
+    is_required: true
```

Comment:
Sorry I didn't notice this earlier, tooling labels should not be required. Please drop this key.

Reply:
Sorry no thats my fault, ill do that immediately